src/TTrigger.h

```diff
@@ -53,6 +53,7 @@ class TMatchState;
 #define REGEX_PROMPT 7
 #define MAX_CAPTURE_GROUPS 33
 
+using NameGroupMatches = QVector<QPair<QString, QString>>;
```

Comment:
:exploding_head: This is a new concept for me - anyone else not get it? https://stackoverflow.com/q/20790932/4805858 might help.

Reply:
Check out https://www.internalpointers.com/post/differences-between-using-and-typedef-modern-c as well

Reply:
I wanted to have this generics more comprehensible.